test/gtest/uct/uct_test.h

```diff
@@ -474,6 +474,11 @@ class test_uct_iface_attrs : public uct_test {
     UCT_TEST_ROCM_MEM_TYPE_TLS, \
     self
 
+#define UCT_TEST_NO_CUDA_MEM_TYPE_TLS \
```

Comment:
reuse in UCT_TEST_TLS:
```
#define UCT_TEST_NO_GPU_MEM_TYPE_TLS  \
  UCT_TEST_NO_SELF_TLS, \
  self
#define UCT_TEST_TLS \
  UCT_TEST_NO_GPU_MEM_TYPE_TLS, \
  UCT_TEST_CUDA_MEM_TYPE_TLS, \
  UCT_TEST_ROCM_MEM_TYPE_TLS
```

Reply:
pls rename it to UCT_TEST_NO_GPU_MEM_TYPE_TLS and remove UCT_TEST_ROCM_MEM_TYPE_TLS from it.
in general the macro names should "GPU" and not "CUDA", and work the same way for Cuda and ROCm